index.js

```diff
@@ -37,4 +47,4 @@ const stringify = (object, type) => {
     }
 }
 
-module.exports = { parse, stringify }
+module.exports = { parse, stringify, removeFileExt }
```

Comment:
Yeah I tried that but it creates files like `input..cson`

Not too hard to change on `ava` but to put effort into removing an optional feature seems counter intuative and it will make the `npm run test` slower as it will need to test on `.cson` and `cson` variables